plugins/harness-ci-cd/src/components/ExecutionList/ExecutionList.tsx

```diff
@@ -1,3 +1,5 @@
+/* eslint-disable import/no-extraneous-dependencies */
+/* eslint-disable @typescript-eslint/no-shadow */
```

Comment:
Rather than disabling let's fix this.

Reply:
Thanks for identifying, they were already fixed removed it from the code, please check the latest commit.